dev/BUILD.yaml

```diff
@@ -13,4 +14,20 @@ packages:
     deps:
       - dev/blowtorch:app
       - dev/gpctl:app
-      - dev/loadgen:app
\ No newline at end of file
+      - dev/loadgen:app
+  - name: dev-utils
+    type: docker
+    deps:
+      - dev/gpctl:app
+      - dev/kubecdl:app
+    argdeps:
+      - imageRepoBase
+    config:
+      buildArgs:
+        VERSION: ${version}
+      dockerfile: leeway.Dockerfile
+      metadata:
+        helm-component: dev-utils
```

Comment:
nit: this package is unlikely to be deployed to a cluster, hence doesn't need this entry

Reply:
Ah, thank you! I was not quite sure how that value is used so I just copy pasted it from another file :) 
Fixed!